cmdstanpy/stanfit.py

```diff
@@ -263,7 +254,11 @@ def stdout_files(self) -> List[str]:
     @property
     def stderr_files(self) -> List[str]:
         """List of paths to CmdStan stderr transcripts."""
-        return self._stderr_files
+        get_logger().warning(
```

Comment:
Does this deprecation imply a release before 1.0, or will be be carrying a deprecation into 1.0?

Reply:
good question.  we should just get rid of them - it's not anything that's exposed to the end user.

Reply:
since all console outputs go to stdout file, it could be renamed -  "msgs", "console_out" ????

Reply:
`out_stream_msgs`? If it's primarily an internal function I don't mind being a little verbose